.github/workflows/create-pr.yml

```diff
@@ -0,0 +1,37 @@
+name: Create Pull Request
+on: 
+# TODO: currently workflow dispatch endpoint does not work. I will investigate
+  repository_dispatch:
+    types: [create-pr]
+  workflow_dispatch:
+jobs:
+  build:
+    name: Create Pull Request
+    env:
+      REPOSITORY_NAME: 'node-versions'
```

Comment:
Let's modify `manifest-generator.ps1` and avoid hardcoded repo name

Reply:
done